src/pages/RandomPage.js

```diff
@@ -1,48 +1,50 @@
 import { useState } from 'react';
 import Card from '../Components/Card';
 import { Button } from '../Components/Button';
-import {ReactComponent as QuestionLogo} from '../Resources/question.svg'
+import { ReactComponent as QuestionLogo } from '../Resources/question.svg';
 
 const RandomPage = ({ characters }) => {
-const [randomID, setRandomID] = useState();
+  const [randomCharacter, setRandomCharacter] = useState();
 
+  const getRandomCharacter = () => {
+    const randomID =
+      Math.floor(Math.random() * (characters.length - 1 - 1)) + 1;
```

Comment:
The following function returns a random integer between 0 and characters.length - 1:
`Math.floor(Math.random() * characters.length)`

Reply:
Hi Thomas,
I have changed it to: Math.floor(Math.random() * characters.length)+1;
The result 0 does not work here as character.IDs start with 1.